src/database/video_emotion_database_no_song.py

```diff
@@ -39,6 +43,16 @@ def find(self, name,
                                    ).sort([('last_updated', -1)]
                                           ).limit(1)[0]
 
+    def find_all_same_id(self, name, video_id):
+        results = []
+        data =  self._db[name].find({'video_id': video_id})
+        for i in data:
+            del i['_id']
+            i['last_updated'] = i['last_updated'].isoformat()
+            results.append(i)
+        return results
+            
+
```

Comment:
You should remove this line. There should only be 1 empty line between functions in a class.